nova/nova/serializers.py

```diff
@@ -28,7 +29,3 @@ def create(self, validated_data):
         validated_data['password'] = make_password(validated_data.get('password'))
 
         return super(UserSerializer, self).create(validated_data)
-
-    def update(self, instance, validated_data):
```

Comment:
Why is this removed

Reply:
Because when I update a user, it was encrypting the encrypted password again. But if the user changes his/her password, it should be encrypted before it is stored, so you are right in that point. I will correct it, thanks!